pkg/api/const.go

```diff
@@ -434,10 +435,23 @@ const (
 const (
 	//AzureADIdentitySystem Identity System
 	AzureADIdentitySystem = "azure_ad"
-	//ADFS Identity System
+	//ADFSIdentitySystem System
 	ADFSIdentitySystem = "adfs"
 )
 
+const (
+	//DependenciesLocationDefault dependencies are in AzurePublic cloud
+	DependenciesLocationDefault = "default"
```

Comment:
Are these constants Azure Stack specific? If so, would it make sense to clarify that by either naming or putting them in an Azure Stack specific field? Something like `AzureStackLocation`.